test/integration/shell_int_test.go

```diff
@@ -288,13 +291,19 @@ func (suite *ShellIntegrationTestSuite) TestNestedShellNotification() {
 
 	cfg, err := config.New()
 	suite.Require().NoError(err)
+	if runtime.GOOS == "darwin" && condition.OnCI() {
+		cfg.Set(subshell.ConfigKeyShell, "zsh") // GitHub actions runs on bash, so override with zsh
```

Comment:
I'd rather we use the `SHELL=zsh` environment variable, as it's more future-proof.

Reply:
We actually cannot do this because `os.Setenv("SHELL")` will affect parallel tests.